### PR TITLE
ZENKO-1010 refactor: zenko charts structure

### PR DESCRIPTION
**Pull request by giacope**
**This change is upgradable.**

So basically the `charts` directory has been renamed to `kubernetes` and it will now look like this:

 └──     kubernetes/
 │  ├────     README.md
 │  ├────     setup-no-orbit.md
 │  ├────     single-node-values.yml
 │  └────     zenko/
 │  │  ├────     Chart.yaml
 │  │  └────     charts/
 │  │  │  └────     backbeat/
 │  │  │  └────     cloudserver/
 │  │  │  └────     grafana/
 │  │  │  └────     mongodb-replicaset/
 │  │  │  └────     prometheus/
 │  │  │  └────     redis-ha/
 │  │  │  └────     s3-data/
 │  │  │  └────     zenko-nfs/
 │  │  │  └────     zenko-queue/
 │  │  │  └────     zenko-quorum/
 │  │  └────     templates/
 │  │  │  ├────     NOTES.txt
 │  │  │  ├────     _helpers.tpl
 │  │  │  ├────     datasource.yaml
 │  │  │  ├────     ingress.yaml
 │  │  │  └────     secrets.yaml
 │  │  └────     values.yaml

With this new format, the command `helm dependency build zenko` will **NEVER** have to be used again. There is no need to build dependencies since they are all present.
It will now be possible to install Zenko with only **ONE** command: `helm install zenko`.

### CODE REVIEW

**Comment by ssalaues** [comment]
@bert-e wait

**Review by ssalaues — APPROVED** [review]
Overall I like this be we should be sure to thoroughly test this before merging. Like have several people test upgrades, fresh installs, update documentation, etc.

File: kubernetes/zenko/charts/zenko-nfs/values.yaml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Default values for zenko-nfs.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+
+enabled: false
```
Comment (ssalaues) [inline kubernetes/zenko/charts/zenko-nfs/values.yaml]: Don't need at this level

**Review by ssalaues — CHANGES_REQUESTED** [review]
One little change

File: kubernetes/zenko/charts/zenko-nfs/values.yaml (hunk 1 above)
Comment (giacope) [reply kubernetes/zenko/charts/zenko-nfs/values.yaml]: It does in order to pass `helm lint --strict`. Changed default value to `true`.

**Review by ssalaues — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

**Pull request merged**